packages/ui/src/containers/eventDetailsContainer/reducer.js

```diff
@@ -0,0 +1,18 @@
+/**
+ * @author Thenmozhi Subramaniam
+ * @description A reducer function that returns the event details
+ * based on the action type passed to it
+ * @param {Object} state
+ * @param {Object} action
+ */
+
+import { GET_EVENTDETAILS } from "../../constants/actions";
+export default function (state = [], action) {
+    console.log('In', action.type)
```

Comment:
Should we store event detail in reducer? I don't think this is required

Reply:
Yes, its not required to be stored in reducer, changed the logic to store the values in state.